### PR TITLE
Exposing numParticipants and numPublishers on Room

### PR DESCRIPTION
**Pull request by davidzhao**
Server will start returning an accurate count of participants and publishers.

This PR is preparing for the future where not all subscribers will have their metadata sent down.

### CODE REVIEW

File: src/room/Room.ts [hunk 1]
```diff
@@ -99,23 +99,13 @@ class Room extends (EventEmitter as new () => TypedEmitter<RoomEventCallbacks>)
...
   localParticipant: LocalParticipant;
 
-  /** room metadata */
-  metadata: string | undefined = undefined;
-
   /** options of room */
   options: InternalRoomOptions;
 
-  private _isRecording: boolean = false;
+  private _room?: RoomModel;
```
Comment (lukasIO) [inline src/room/Room.ts:108]: nit: personally not a huge fan of `room._room`, maybe calling it `info` instead would be an option?

File: src/proto/google/protobuf/timestamp.ts [hunk 2]
```diff
@@ -127,31 +127,22 @@ export const Timestamp = {
   },
 
   decode(input: _m0.Reader | Uint8Array, length?: number): Timestamp {
-    const reader = input instanceof _m0.Reader ? input : _m0.Reader.create(input);
+    const reader = input instanceof _m0.Reader ? input : new _m0.Reader(input);
```
Comment (lukasIO) [inline src/proto/google/protobuf/timestamp.ts:130]: we keep having diffs like this from time to time, wondering whether there's a difference in the ts-proto versions we're using...

**Review by Ocupe — APPROVED** [review]

File: src/proto/google/protobuf/timestamp.ts (hunk 2 above)
Comment (davidzhao) [reply src/proto/google/protobuf/timestamp.ts:130]: It appears so. I just did a `yarn install` and it should be reverted back.

File: src/room/Room.ts (hunk 1 above)
Comment (davidzhao) [reply src/room/Room.ts]: I'll update it to `roomInfo`

**Pull request merged**